bot/resources/evergreen/python_facts.txt

```diff
@@ -0,0 +1,3 @@
+Python was named after Monty Python, which Guido van Rossum likes.
```

Comment:
What is `Monty Python`? Mention that in here.

Reply:
ok

Reply:
Fixed